src/modules/api/user/link/link.controller.ts

```diff
@@ -1,7 +1,21 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Get, Param } from '@nestjs/common';
 import { LinkService } from './link.service';
 
 @Controller('users/:userId/links')
 export class LinkController {
-  constructor(private linkService: LinkService) {}
+  constructor(private readonly linkService: LinkService) {}
+  @Get()
+  async getAllLinksForUserId(@Param('userId') userId:string) {
+    const links = await this.linkService.getAllLinksForUser(userId);
+    if (links.length) {
```

Comment:
if(links?.length) is better way to do it.
You got to use Optional Chaining to make sure that this doesn't throw any error

Reply:
When there are no matches find() returns [] so i went with the length